### DISCUSSION TITLE
Redux with next js API routes

### DISCUSSION THREAD

**Discussion by syed1994** [vercel/next.js#11827] [Help]
Hi everyone,
I am new to next js.i have been integrating a custom node server with next app .

As a follow up I have tried to integrate next js and am quite successful with react components inside pages folder. But I struck with files inside pages/api folder where next provides support to build APIs.I cannot able to get redux state into those APIs.

Can anyone help me out on this. Thanks in advance.

**Comment by SnorreFossland** [comment]
I have the same problem....
